VideoLocker/src/main/java/org/edx/mobile/module/analytics/ISegmentImpl.java

```diff
@@ -189,7 +190,8 @@ public Properties trackVideoSeek(String videoId,
             skipInterval = formatDoubleValue(skipInterval, 3);
             aEvent.data.putValue(Keys.OLD_TIME, oldTime);
             aEvent.data.putValue(Keys.NEW_TIME, newTime);
-            aEvent.data.putValue(Keys.SEEK_TYPE, Values.SKIP);
+            if (skipSeek){ aEvent.data.putValue(Keys.SEEK_TYPE, Values.SKIP);}
```

Comment:
Instead of a boolean, maybe it should be an enum? I guess I just think skipSeek is kind of a confusing name?

Then it would be like SKIP and SLIDE, matching up with the values?